docs/examples/State Estimation Example.ipynb

```diff
@@ -36,10 +36,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 4,
    "id": "ae11dc9a",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
```

Comment:
you get errors... that shouldn't be, right? did you install all optional dependencies required to run the examples? you may need to create a new optional dependency group `[examples]` in the `pyproject.toml` file (just like how `[dev]` is defined)